core/domain/suggestion_services.py

```diff
@@ -550,6 +579,64 @@ def get_submitted_suggestions(user_id, suggestion_type):
     ])
 
 
+def get_user_scoring_from_model(user_scoring_model):
+    """Converts the given UserContributionScoringModel to a
```

Comment:
Should this be named ``convert_...`` or ``create_..._from_...`` to match what it's doing?